plugins/mailchimp.md

```diff
@@ -0,0 +1,26 @@
+# Mailchimp integration
+
+Mautic can send leads to Mailchimp upon some lead's action or when it gain some point limit. Mailchimp can be also used as an email sending system. This option is not part of the plugin though so it will not be covered on this page.
```

Comment:
`Mailchimp can be also used as an email sending system` may confuse some as it's not Mailchimp but Mandrill which is a service of Mailchimp.  I'm not sure I would mention that at all as it's not related to the Mailchimp integration.